nms/app/packages/magmalte/app/views/metrics/LteMetricsData.js

```diff
@@ -0,0 +1,783 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+export type MetricsDetail = {
+  name: string,
+  promQL: string,
```

Comment:
I'm pretty skeptical of the use of having a promQL field given that the vast majority of these are just going to repeat the name, and then a small portion of them are going to have a simple label matcher appended to it. What would be better would be to have a metric name, then list the available labels that exist with that metric. Perhaps a little instructional paragraph about how to query for specific labels would be nice.

Reply:
I'm not exposing this to the user in the table, instead it is to pull up the grafana explore iframe when the table row is expanded

Reply:
OH NEVERMIND. I didn't realize you're using this to generate grafana dashboards on the fly. That's super cool if you can get it to work